packs/data/extinction-curse-bestiary.db/doblagub.json

```
@@ -1610,11 +1702,11 @@
                 "damageRolls": {
                     "unnje74zfapu55tanl47": {
                         "damage": "",
-                        "damageType": "Pull10feet"
+                        "damageType": "chaotic"
```

```suggestion
                        "damageType": "bludgeoning"
```
It doesn't do damage anyway, but bludgeoning fits a tongue attack more.